substrate/frame/revive/src/precompiles/builtin/testdata/901-blake2_256.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "Input": "bbe04a1d00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000",
+    "Expected": "03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314",
+    "Name": "vector 1",
+    "Gas": 8000000,
+    "NoBenchmark": false
+  }
+]
```

Comment:
@athei I could use some guidance on how you originally came up with these files, I've assembled it manually for now. Not sure how to decide on `Gas` here, I just put some generous value here now.